### PR TITLE
Update form.rst

### PR DESCRIPTION
**Pull request by Ferikl**
Add example how to disable CSRF in Form

### CODE REVIEW

**Comment by u-mulder** [comment]
Replace `function.:` with `function:`.

**Review by royklutman — APPROVED** [review]

File: components/form.rst [hunk 1]
```diff
@@ -153,6 +153,13 @@ the CSRF generator and validated when binding the form.
 
         $csrfStorage = new NativeSessionTokenStorage();
         // ...
+        
+.. tip::
+    If you want to disable CSRF on your form, you can give specific options to your form builder through the createBuilder() function:
+        
+        use Symfony\\Component\\Form\\Extension\\Core\\Type\\FormType
```
Comment (nicolas-grekas) [inline components/form.rst]: missing `;` at end of line (same below)

File: components/form.rst [hunk 2]
```diff
@@ -153,6 +153,13 @@ the CSRF generator and validated when binding the form.
 
         $csrfStorage = new NativeSessionTokenStorage();
         // ...
+        
+.. tip::
+    If you want to disable CSRF on your form, you can give specific options to your form builder through the createBuilder() function:
```
Comment (nicolas-grekas) [inline components/form.rst]: missing double backticks around createBuilder()

**Comment by xabbuh** [comment]
Thank you @Ferikl. I have made some minor tweaks to your changes in 33933770bdfc20ec29658e7764cd0583bfebaa87 and merge your PR into the `2.8` branch.

**Pull request closed without merging**